CMakeLists.txt

```diff
@@ -2265,6 +2272,9 @@ if(NOT CURL_DISABLE_INSTALL)
       OWNER_READ OWNER_WRITE OWNER_EXECUTE
       GROUP_READ GROUP_EXECUTE
       WORLD_READ WORLD_EXECUTE)
+
+  set(CPACK_GENERATOR "TGZ")
```

Comment:
Does the `TGZ` generator work for non-Unix targets?

Ref: https://cmake.org/cmake/help/book/mastering-cmake/chapter/Packaging%20With%20CPack.html#options-added-by-cpack

Reply:
It actually works. (The documentation seems need be updated) And this variable is just a default option when `-G` is not provided for cpack.